Gems/Blast/Code/Source/Components/BlastFamilyComponent.cpp

```diff
@@ -71,33 +71,33 @@ namespace Blast
                     "Radial Damage", &BlastFamilyDamageRequests::RadialDamage,
                     {{{"position", "The global position of the damage's hit."},
                       {"minRadius", "Damages all chunks/bonds that are in the range [0, minRadius] with full damage.",
-                       AZ::BehaviorDefaultValuePtr(aznew AZ::BehaviorDefaultValue(static_cast<float>(0.0)))},
+                       AZ::BehaviorDefaultValuePtr(aznew AZ::BehaviorDefaultValueTyped(static_cast<float>(0.0)))},
```

Comment:
These functions can be switched to just use the `behaviorContext->MakeDefaultValue` function.
That way the `AZ::BehaviorDefaultTyped` class never needs to be interacted with directly.